### ISSUE TITLE
Random.natIn that is inclusive of upper value

### ISSUE THREAD

**Issue by ceedubs** [unisonweb/base#170]
I would like to generate a `Nat` with an inclusive upper value of `maxNat`. Is there a straightforward way to do that currently? Since `natIn lower upper` is exclusive on `upper`, you can't do `natIn lower maxNat`. For cases _other than_ `maxNat`, you could do `natIn lower (increment upper)`, but this will overflow and give an error if `upper` is `maxNat`.

See also #169.

**Comment by pchiusano** [comment]
@ceedubs you can use `Random.nat` to generate `Nat` in the full range 0 to maxNat, does that help?

**Comment by ceedubs** [comment]
@pchiusano yes, but it's a bit annoying. In my current use-case the upper bound is passed in as an argument to the function, so I'll have to check whether the upper bound is `maxNat` and use `nat` if it is and `natIn` if it is not. If the lower bound were also passed in as an argument I think that I'd still have the same trouble.
